multi-era/rust/src/utils.rs

```diff
@@ -0,0 +1,102 @@
+        let _block = MultiEraBlock::from_explicit_network_cbor_bytes(&bytes).unwrap();
+    }
+
+    #[test]
+    fn babbage_network_block_parse() {
+        let bytes = hex::decode("820685828a1a0002a1d21a00360d985820f93e682d5b91a94d8660e748aef229c19cb285bfb9830db48941d6a78183d81f58209691ed9d98a5b79d5bc46c4496a6dba7e103f668f525c8349b6b92676cb3eae45820ea49e4652c460b9ee6daafefc999ca667fbe5eb5d7a7aeabbdff6fe19c1a3c9f825840c448c14556645bfb1c2de174b4f375bd8e5c27a41f26661b17d05ab8d30e9f2527681555316ce65a7e4dfb3cf83f57144a95df44d1fa67293c31f0ab15c4f4b55850c15e1be17e969159dd7ea5df78af4154767e6c13e4f41cf67d9bc305f6f4d0e1ec56df71df7e00ab51d9f7de9bdefc88e60a155f4af7225007146c38cff4d5ca7a8067214baba73aa5a2f726da72360904582029571d16f081709b3c48651860077bebf9340abb3fc7133443c54f1f5a5edcf18458200481896957dda8d02d825448eb22aad5e106114d924f49923b2fd68e6b3254d3000058403b0db934233f93c43de54a63f8be99bd21da431dcea996a01213ada37e66c6a3930a95399ce083875ba95ffa58cb8ca9b83c92f7d440adb968d8569d74fe90018207005901c0c8e3bb3da1d72fa8a736a6a66bec894083a7f74b388181c54b8faaaf7ccd020acd7a7fbde4a2505f6fbfa907a8bca491219713841720e122a94a057ad065850e88894f7b7985335bd2e3c6bbed5a3fa7fd526a8afafd78fd04dfaef371ad55d0d3d0f1707db5e6773f3ccb896d9f81b1428c3dbcf693c54580f91bb93364272349c08980f71e74e74f8d7863a29cee3fbd821642c14285d8198c68fdbc0a193bf63d97c7a5481ce7da17c97f43a7b98d0cad8c3af26ae4f9ac695fab06ca34685ec57339673d084e0417f844747025f4f5b42cbd883159a3d52dab9dafdfb37b0136757b67ec4e1a6768c43cee5937b81e03224aa6909a9a3254cdf33887914d67723f9a5a4505ac41b658cec10d6c2949e3d3c58bc7a27b64eb8407cbdad7806db704e7ee172405ddaa634119f878541d2f19674dcd0aebf4e6e1091635ce177489742c28d61a4698edaea3364b92b03a73838ca24ce83913727154dae3c2ee4a331997dacd155753f014bc98cfa25b8d84c808654852b5e1e74e251651e30063a6451a676795aa421ea59daa711a7ff0d4140de86bc89969f717d461690c341da4cd440818205961ed9cc9970f52b10ac1b1c3ed3e0d0170582daec84ef2068080a080").unwrap();
+        let _block = MultiEraBlock::from_explicit_network_cbor_bytes(&bytes).unwrap();
+    }
+
+    // TODO: Conway test
```

Comment:
why conway is still todo if we have the blocks already?

Reply:
When I wrote the tests for some reason I wasn't able to get cardano-net-fetcher to connect even using the same commands as before. I can try it again now.

Reply:
Yeah I can't seem to get it to work anymore. The other networks are fine. It just closes without errors regardless of what I pass into `--since` (even omitted)

Reply:
```
Block #457890, point: c22584bc3714c23f798fa65f7ed0b7e5fec77155574968b5995b5837e41949cd@9215930, raw cbor hex: 820785828a1a0006fca21a008c9fba5820a274eb87d9c4af6296f98990309b8cfb55628760672acfe9b15cd038a4931cf858205a80dddc5e5856c3d287b51ab8e2308545aee93c0395b20a037128d7a08d4f56582044c87328bfdf1a91f1326ab354e45499d9eceb368ce62207e7b8f66f3924acd48258408f1a5a844fdaf133d4b78975847d3387aff675004f3efcca24450b76b3e441f0590f1399961d7e8a9e9e2c948e6357c943b6ff6de9986f91e0f6e1608e4cd7845850e125baae13a10225da2c80cf4af46f03e7353d33c298c0539850d3ee94a8370df11b42e6d98b8a8fec5cc7e29ab994e78b41dd891ac4f2d656fe6a76884d51f8491ede37900e199488b82b9dc0fbd40204582029571d16f081709b3c48651860077bebf9340abb3fc7133443c54f1f5a5edcf18458205d9a69836d614e6e1524118c14c53dd5699e5fc92b0d1aa6dcc44c1ccbd5b0af001845584035af87da142c87c0fc065d83371d44b28269a18e2f19380c0b4167913ac2820001ce9be51711de2370a1e203ee32fefc2d1bca66916de0c2ea3b0fb65391690d8209005901c08e0a4a2b2a70b11b7abd7641f9778a7bf4538c41c4ac1e1630a941653c4e2f6392b53fa5849367ed5bf44fa1b47b6ad9c5e0fd9fb477fd1f43e531e72cea740245521d448973c583919b9afdbef900a73ca2c52b914981248536131d22aae2bde34ba27f98e09fd34f87be64ff4cdf07d9f34059bc135b3e835bd7f072c52436e7c191252edaf63a8b31f5059dc140caaf342dccf303b21585dc67beb5c45e10301deffc9196b740e84a266f14c8218401382fe683af05f3d98667a8193b2335e40fb07ffee31d444660862bb3d06aa77212bec99eb50b1434de613dbd3014902de2225fad4ca1b57aa96d3ee68ecb364d5d721f6b2d9546e2dbca1ee70a581103ccd2bea94eae745dd97abd077c3bbed1c8a9fe75963b1f0de8f93a378e052f1281f583f84a1873a8de298bc3949592de8edf4ac86d4dce953ca4efe31549a744192b7740257ce0e7cf7029ba314539b2bf7b2434d52544e61d5643ddb78f92c8632c9bf266bb9ca5e85eb61864264d0f2ba9bca7f25f041f26321833c56d7d6ee5baca46c3e5e70708b9f898f329706e564d9b385872b2a7e04f15e8b7fb6b5bcf702352028962a30a792766092f6bbba5acbc729183f371cb3c45a9b6f8a38080a080
```

Reply:
or like this: 
```
cargo run --bin cardano-net-fetcher -- --network sancho --relay-host sanchonet-node.world.dev.cardano.org --relay-port 30004 --since 8986733,7c2f48f06dc110e7935c3cae6504772936b9322d0b857342c718d4eda4f715f1
```

Reply:
got some newer points from blockfrost and iog discord

Reply:
Weird. That `--since` works. The `--since` from the slack thread isn't working (`7257117,d156058438a0be60eae635dbcbf70bfc6dc582fe28141fc7c3061000262dfd67`) from where the Conway blocks started before. The above one works. I wonder why? That's how I was checking them last week.